addons/web_editor/static/src/scss/wysiwyg.scss

```diff
@@ -403,8 +404,10 @@ body:not(.editor_has_snippets) {
         z-index: 1056;
     }
 }
-@media only screen and (max-width: 767px) {
+
+@include media-breakpoint-down(md) {
     .oe-toolbar {
+        position: static;
```

Comment:
Can't we use `position-static position-md-absolute` classes combination?

Reply:
The problem if I do that, is that the toolbar won't be sticky on mobile (@include o-position-sticky($top: 0px);) anymore in modules with notebook like project.

Reply:
yes, ok, thanks